cdc/redo/manager.go

```diff
@@ -447,7 +471,7 @@ func (m *ManagerImpl) flushLog(ctx context.Context, handleErr func(err error)) {
 		defer atomic.StoreInt64(&m.flushing, 0)
```

Comment:
why a go routine is needed here?

Reply:
flush should be performed in an individual goroutine. It's same with the master branch.